src/url-state-machine.js

```diff
@@ -435,13 +434,9 @@ function findLongestZeroSequence(arr) {
   // if trailing zeros
   if (currLen > maxLen) {
     maxIdx = currStart;
```

Comment:
Could also make this return currStart directly. That's probably how I'd write it down.

Reply:
Agreed. I'll push a commit that does that and merge.